README.md

```diff
@@ -1278,131 +1259,25 @@ terragrunt = {
-
-          # Tell Terraform to do locking using DynamoDB. Terragrunt will automatically create this table for you if
-          # it doesn't already exist.
-          lock_table = "my-lock-table"
-        }
-      }
-    }
-    ```
-
-    **NOTE**: We recommend using a completely new lock table name and NOT reusing the lock table from older versions of
-    Terragrunt, as that older table had a different structure than what Terraform expects, and Terragrunt will not
-    automatically recreate it.
-
-    If you would like Terraform to automatically retry locks like Terragrunt did (this is particularly useful when
-    running Terraform as part of an automated script, such as a CI build), you use an `extra_arguments` block:
-
-    ```hcl
-    # terraform.tfvars
-    terragrunt = {
-      remote_state {
-        backend = "s3"
-        config {
-          bucket  = "my-terraform-state"
-          key     = "${path_relative_to_include()}/terraform.tfstate"
-          region  = "us-east-1"
-          encrypt = true
-
-          # Tell Terraform to do locking using DynamoDB. Terragrunt will automatically create this table for you if
-          # it doesn't already exist.
-          lock_table = "my-lock-table"
-        }
-      }
-
-      terraform {
-        # Force Terraform to keep trying to acquire a lock for up to 20 minutes if someone else already has the lock
-        extra_arguments "retry_lock" {
-          commands = [
-            "init",
-            "apply",
-            "refresh",
-            "import",
-            "plan",
-            "taint",
-            "untaint"
-          ]
-
-          arguments = [
-            "-lock-timeout=20m"
-          ]
-        }
-      }
-    }
-    ```
+Terragrunt figures out the path to its config file according to the following rules:		
+ 		
+ 1. The value of the `--terragrunt-config` command-line option, if specified.
+ 1. The value of the `TERRAGRUNT_CONFIG` environment variable, if defined.
+ 1. A `terraform.tfvars` file in the current working directory, if it exists.
+ 1. If none of these are found, exit with an error.
+ 		
+ The `--terragrunt-config` parameter is only used by Terragrunt and has no effect on which variable files are loaded
+ by Terraform. Terraform will automatically read variables from a file named `terraform.tfvars`, but if you want it
+ to read variables from some other .tfvars file, you must pass it in using the `--var-file` argument:		
+ 
+ ```bash
+  terragrunt plan --terragrunt-config example.tfvars --var-file example.tfvars		
+ ```
 
+:exclamation: **Note:** Terragrunt `v0.11.x` and earlier defined the config in a `.terragrunt` file. **The `.terragrunt`
+format is now deprecated**! You will get a warning in your logs every time you run Terragrunt with a 
+`.terragrunt` file, and we will eventually stop supporting this older format. We recommend following
+the migration guide ASAP: [Upgrading to Terragrunt 0.12.x](migration_guides/upgrading_to_terragrunt_0.12.x.md).
```

Comment:
Rather than an `exclamation:` here, I'd prefer to see a header like "Previous Versions of Terragrunt". Something like this:

#### Previous Versions of Terragrunt

Terragrunt v0.11.x and earlier defined the config in a `.terragrunt` file. Note that **the `.terragrunt` format is now deprecated**. You will get a warning in your logs every time you run Terragrunt with a `.terragrunt` file, and we will eventually stop supporting this older format. 

#### Migrating from Previous Versions of Terragrunt

After we released support for Terraform 0.9.x, we wrote a guide on [Upgrading to Terragrunt 0.12.x](migration_guides/upgrading_to_terragrunt_0.12.x.md).